resources/css/bem/team-summary.less

```diff
@@ -9,7 +9,7 @@
   gap: 20px;
   grid-template-columns: 1fr;
   align-items: start;
-  max-height: calc(70 * var(--vh));
+  max-height: 400px;
```

Comment:
it can be `70svh` since `svh` uses the smallest viewport size and is widely supported now